### DISCUSSION TITLE
Collected Fdroid Reposorities (in case you want to add more)

### DISCUSSION THREAD

**Discussion by mondlicht-und-sterne** [NeoApplications/Neo-Store#107] [Ideas]
2br-2b
https://raw.githubusercontent.com/2br-2b/Fdroid-repo/master/fdroid/repo/

Alefvanoon's Repo
https://repo.alefvanoon.xyz/fdroid/repo/?fingerprint=04DF198F553069C7BE60F057AE12000E99F7700DA895CC1CE2EB11DC871581F1

Antox
https://pkg.tox.chat/fdroid/repo

app.usb0.net
https://app.usb0.net/repo

Bitwarden
https://mobileapp.bitwarden.com/fdroid/repo

Briar
https://briarproject.org/fdroid/repo

Bromite
https://fdroid.bromite.org/fdroid/repo

c:geo
https://fdroid.cgeo.org

c:geo legacy
https://fdroid.cgeo.org/legacy

c:geo nightly
https://fdroid.cgeo.org/nightly

Calyx Institute
https://calyxos.gitlab.io/calyx-fdroid-repo/fdroid/repo

CCC event repo
https://fdroid.gitlab.io/ccc/fdroid/repo/

Collabora Office
https://www.collaboraoffice.com/downloads/fdroid/repo

Corona Contact Tracing Germany
https://bubu1.eu/cctg/fdroid/repo/?fingerprint=f3f30b6d212d84aea604c3df00e9e4d4a39194a33bf6ec58db53af0ac4b41bec

Cryptomator
https://static.cryptomator.org/android/fdroid/repo?fingerprint=F7C3EC3B0D588D3CB52983E9EB1A7421C93D4339A286398E71D7B651E8D8ECDD

cutthecord
https://fdroid.a3.pm/seabear/repo

Devaaa
https://qrcrypto.ch/fdroid/repo

DivestOS Official
https://divestos.org/fdroid/official/?fingerprint=E4BE8D6ABFA4D9D4FEEF03CDDA7FF62A73FD64B75566F6DD4E5E577550BE8467

DivestOS Unofficial
https://divestos.org/fdroid/unofficial/?fingerprint=A18CDB92F40EBFBBF778A54FD12DBD74D90F1490CB9EF2CC6C7E682DD556855D

Ember’s Unofficial uGet
https://uget.heartshine.xyz/fdroid/repo

Eutopia
https://eutopia.cz/fdroid/repo

Eutopia experimental
https://eutopia.cz/experimental/fdroid/repo

F-Droid
https://f-droid.org/repo

F-Droid Archive
https://f-droid.org/archive

F-Droid Classic
https://bubu1.eu/fdroidclassic/fdroid/repo?fingerprint=5187CFD99F084FFAB2AD60D9D10B39203B89A46DD4862397FE1B1A4F3D46627A
more info: https://gitlab.com/Bubu/fdroidclassic/-/blob/master/README.md#f-droid-classic

F-Droid nightly
https://gitlab.com/fdroid/fdroidclient-nightly/raw/master/fdroid/repo

Firefox unofficial
https://rfc2822.gitlab.io/fdroid-firefox/fdroid/repo

Fedilab
https://fdroid.fedilab.app/repo/?fingerprint=11F0A69910A4280E2CD3CCC3146337D006BE539B18E1A9FEACE15FF757A94FEB

Fluffychat
https://fdroid.fluffychat.im/?fingerprint=8E2637AEF6697CC6DD486AF044A6EE45B1A742AE3EF56566E748CDE8BC65C1FB

Fluffychat Nightly
https://nightly.fdroid.fluffychat.im?fingerprint=21A469657300576478B623DF99D8EB889A80BCD939ACA60A4074741BEAEC397D

Frostnerd’s Nebulo
https://fdroid.frostnerd.com/fdroid/repo

Funkwhale
https://fdroid.funkwhale.audio/?fingerprint=103063BC7189C91CE727DBF8266B07662518096E1686B6A088253933A3D0788F

gnu-taler
https://gitlab.com/gnu-taler/fdroid-repo-nightly/raw/master/fdroid/repo

gsantner’s F-Droid repository
https://gsantner.gitlab.io/fdroid/repo

Guardian Project
https://guardianproject.info/fdroid/repo

Guardian Project Archive
https://guardianproject.info/fdroid/archive

Haagch
https://haagch.frickel.club/files/fdroid/repo/

insporation
https://jhass.github.io/insporation/fdroid/repo?fingerprint=EC792A58B39DF9FBB466FB100E30E3842F229FDBC6E28D32C417F6A5B30ECCAE

InvizBox 2 Administration App
https://update.invizbox.com/fdroid/repo?fingerprint=FFA1A810B48608135EEFD7C692F2306172FB578549C4111A70266374A119D189

IzzyOnDroid
https://apt.izzysoft.de/fdroid/repo

I2P
https://fdroid.i2p.io/repo

I2P Archive
https://f-droid.i2p.io/archive

J2ghz
https://j2ghz.github.io/repo

J2ghz Archive
https://fdroid.j2ghz.com/archive

Julian Klode
https://jak-linux.org/fdroid/repo

Juwelierkassa
https://juwelierkassa.at/fdroid/repo?fingerprint=930270248A02202D517518474D0C0215070A537DE3681D8721D84A9B1693915E

kaffeemitkoffein
https://kaffeemitkoffein.de/fdroid/repo

Kali NetHunter
https://store.nethunter.com/repo

KDE
https://cdn.kde.org/android/fdroid/repo
Repo info: https://community.kde.org/Android/FDroid

Kontalk
https://www.kontalk.org/fdroid/repo

Kuschku
https://repo.kuschku.de/fdroid/repo/

Ltt.rs for Android
https://ltt.rs/fdroid/repo?fingerprint=9C2E57C85C279E5E1A427F6E87927FC1E2278F62D61D7FCEFDE9346E568CCF86

Libre Church
https://repo.librechurch.org/fdroid

LibRetro / RetroArch
https://store.nethunter.com/repo

Lubl
https://f.lubl.de/repo

Luca App
https://lucaapp.gitlab.io/fdroid-repository/fdroid/repo
Sourcecode https://gitlab.com/lucaapp/fdroid-repository

Metatrans Apps
https://metatransapps.com/fdroid/repo/

microG
https://microg.org/fdroid/repo

microG archive
https://microg.org/fdroid/archive

Molly (Signal Fork)
https://molly.im/fdroid/

Molly-FOSS
https://molly.im/fdroid/foss/repo/?fingerprint=5198DAEF37FC23C14D5EE32305B2AF45787BD7DF2034DE33AD302BDB3446DF74

Monerujo
https://f-droid.monerujo.io/fdroid/repo

Nailyk
https://releases.nailyk.fr/repo/

Nanodroid
https://nanolx.org/fdroid/repo

Netsyms Technologies
https://repo.netsyms.com/fdroid/repo

NewPipe
https://archive.newpipe.net/fdroid/repo

Nit Language
http://nitlanguage.org/fdroid

obfusk's f-droid repo
https://obfusk.dev/fdroid/repo?fingerprint=2A21B7FFC93B878724B1991C05DAE113C72B93A556C193F49B5D3342884798B7

Omega Launcher
https://raw.githubusercontent.com/rafaelvenancio98/fdroid/master/fdroid/repo

Onosendai
https://onosendai.herokuapp.com/repo

PartidoPirata
https://fdroid.partidopirata.com.ar/fdroid/repo/

Patched Applications (Langis)
https://thecapslock.gitlab.io/fdroid-patched-apps/fdroid/repo/?fingerprint=313D9E6E789FF4E8E2D687AAE31EEF576050003ED67963301821AC6D3763E3AC

Pili
https://pili.qi0.de/fdroid/repo/

pilight
https://android.pilight.org

Platschi On Droid
https://platschi.net/fdroid/repo

Platschi On Droid Archive
https://platschi.net/fdroid/Archive

Project Insanity
https://onny.project-insanity.org/fdroid/repo

Rakshazi
https://fdroid.rakshazi.me/repo/

Rami
https://fdroid.rami.io/fdroid/repo/

RedReader alpha
https://redreader.org/fdroid

RetroArch
https://fdroid.libretro.com/repo

RetroArch Archive
https://fdroid.libretro.com/archive

RWTH Aachen Uni
https://app.rwth-aachen.de/fdroid

Samourai bitcoin wallet
https://repo.samourai.io/fdroid/repo?fingerprint=5318AFA280284855CF5D0027AA54517769F461D735980B1FB0854CEAE8E072A5

SecondWind
https://guardianproject-wind.s3.amazonaws.com/fdroid/repo/

Session
https://fdroid.getsession.org/fdroid/repo

Silkevicious Apk Repo
https://codeberg.org/silkevicious/apkrepo/raw/branch/master/fdroid/repo?fingerprint=DFDB0A58E78704CAEB609389B81AB2BE6A090662F860635D760E76ACBC700AF8
Sourcecode https://codeberg.org/silkevicious/apkrepo

Simlar
https://raw.githubusercontent.com/simlar/simlar-fdroid-repo/master/fdroid/repo

Spinster
https://fdroid.feministwiki.org/fdroid/repo/

StoryMaker
https://s3.amazonaws.com/storymaker/fdroid/repo

Submarine
https://submarine.strangled.net/fdroid/repo/?fingerprint=3DB2D09F573245E005B6A54EB5EBF6E6A351D6A5161CFEEFF1442C939597C31B

Tagesschau
https://service.tagesschau.de/app/repo

Tiny Tiny RSS
https://srv.tt-rss.org/fdroid/repo

Torsten Grote’s personal repo
https://grobox.de/fdroid/repo

TwinHelix - Signal-FOSS
https://fdroid.twinhelix.com/fdroid/repo?fingerprint=7B03B0232209B21B10A30A63897D3C6BCA4F58FE29BC3477E8E3D8CF8E304028

Umbrella
https://secfirst.org/fdroid/repo

Ungoogled Chromium
https://www.droidware.info/fdroid/repo

Unofficial Anime/Manga
https://animanga.efreakbnc.net/fdroid/repo/

Unofficial F-Droid repo for various apps
https://rfc2822.gitlab.io/fdroid-firefox/fdroid/repo

Unofficial Protonmail
https://unofficial-protonmail-repository.gitlab.io/unofficial-protonmail-repository/fdroid/repo

Unofficial Tachiyomi extensions
https://tachi.efreakbnc.net/fdroid/repo

VideLibri
https://fdroid.videlibri.de/repo

WiFi AdBlocker
https://app.usb0.net/repo

WOZ
https://fdroid.woz.ch/fdroid/repo?fingerprint=F19BB95E4DD3B74906E2EF0BA41EA6F34AB82C44F78E1B9505FE207DD5E60CC2

Yves Fischer
https://www.xapek.org/~yvesf/public/fdroid/repo

Zevvle
https://fdroid.zevvle.com/?fingerprint=B6E80E4DB400B5D16A2CD5D1B2D476D1755D9777CE135E8359D166A70BCD12A9

**Comment by yoshimo** [comment]
Stack Wallet uses
https://fdroid.stackwallet.com/?fingerprint=764B4262F75750A5F620A205CEE2886F18635FBDA18DF40758F5A1A45A950F84

**Reply by machiav3lli** [reply]
Done!

**Comment by machiav3lli** [comment]
@mondlicht-und-sterne Thanks! I just noticed this, seems it went under the many notifications at that time.

I'll see what can be used of those, but it would really helpful if you can clean up the list of repos already existing in the app.

**Comment by yoshimo** [comment]
@mondlicht-und-sterne  while you are at it, cross check with https://android.izzysoft.de/articles/named/list-of-fdroid-repos?lang=en

**Comment by dreirund** [comment]
> CCC event repo
> https://fdroid.gitlab.io/ccc/fdroid/repo/

This seems to be no longer maintained (pretty outdated, even the 36C3 app is older than the latest 36C3 app that was released on f-droid.org).

I recommend to delete it from the list.

*Also: Does anyone know how to contact the maintainers of that repository, in order to get it taken down/ have displayed a big warning that it is outdated, or to have it updated?*

Regards!

**Comment by yoshimo** [comment]
I think the diff between suggested repos and the ones we already have comes down to:

```
http://bdf2wcxujkg6qqff.onion/fdroid/repo
http://nitlanguage.org/fdroid
https://android.pilight.org
https://animanga.efreakbnc.net/fdroid/repo/
https://apk.vanderwarker.dev/fdroid/repo
https://app.rwth-aachen.de/fdroid
https://app.simplex.chat/fdroid/repo
https://app.usb0.net/repo
https://bubu1.eu/cctg/fdroid/repo/
https://bubu1.eu/fdroidclassic/fdroid/repo
https://calyxos.gitlab.io/calyx-fdroid-repo/fdroid/repo
https://codeberg.org/florian-obernberger/fdroid-repo/raw/branch/main/repo
https://codeberg.org/silkevicious/apkrepo/raw/branch/master/fdroid/repo
https://codeberg.org/timedin/fdroid-repo/raw/branch/main/repo
https://divestos.org/fdroid/unofficial/
https://ds.jugendhacker.de/fdroid/repo
https://eutopia.cz/experimental/fdroid/repo
https://eutopia.cz/fdroid/repo
https://eyedeekay.github.io/fdroid/repo
https://f-droid.i2p.io/archive
https://f-droid.i2p.io/repo/
https://f-droid.org/archive
https://f.lubl.de/repo
https://fdroid.a3.pm/seabear/repo
https://fdroid.aniyomi.org/
https://fdroid.beocode.eu/fdroid/repo/
https://fdroid.cgeo.org/legacy
https://fdroid.feministwiki.org/fdroid/repo/
https://fdroid.fluffychat.im/
https://fdroid.i2p.io/repo
https://fdroid.j2ghz.com/archive
https://fdroid.krombel.de/riot-dev-fdroid/fdroid/repo
https://fdroid.krombel.de/riot-stable/fdroid/repo
https://fdroid.libretro.com/archive
https://fdroid.ltheinrich.de/
https://fdroid.metatransapps.com/fdroid/repo/
https://fdroid.nordgedanken.dev/repos/CeqjCgrW_4yY_px59cntLkkAov3QRlwQ/repo
https://fdroid.partidopirata.com.ar/fdroid/repo/
https://fdroid.pixelfed.net/fdroid/repo/
https://fdroid.rakshazi.me/repo/
https://fdroid.rami.io/fdroid/repo/
https://fdroid.videlibri.de/repo
https://fdroid.woz.ch/fdroid/repo
https://fdroid.zevvle.com/
https://freeyourgadget.codeberg.page/fdroid/repo/
https://github.com/onionshare/onionshare-android-nightly/raw/master/fdroid/repo
https://gitlab.com/fdroid/fdroidclient-nightly/raw/master/fdroid/repo
https://gitlab.com/gnu-taler/fdroid-repo-nightly/raw/master/fdroid/repo
https://gsantner.gitlab.io/fdroid/repo
https://haagch.frickel.club/files/fdroid/repo/
https://j2ghz.github.io/repo
https://jhass.github.io/insporation/fdroid/repo/
https://juwelierkassa.at/fdroid/repo
https://kaffeemitkoffein.de/fdroid/repo
https://ltt.rs/fdroid/repo
https://lublin.se/fdroid/repo
https://lucaapp.gitlab.io/fdroid-repository/fdroid/repo
https://metatransapps.com/fdroid/repo/
https://molly.im/fdroid/foss/repo/
https://nightly.fdroid.fluffychat.im
https://nitlanguage.org/fdroid
https://obfusk.dev/fdroid/repo
https://onny.project-insanity.org/fdroid/repo
https://onosendai.herokuapp.com/repo
https://pili.qi0.de/fdroid/repo/
https://pkg.tox.chat/fdroid/repo
https://platschi.net/fdroid/Archive
https://platschi.net/fdroid/repo
https://qrcrypto.ch/fdroid/repo
https://raw.githubusercontent.com/2br-2b/Fdroid-repo/master/fdroid/repo/
https://raw.githubusercontent.com/MM2-0/fdroid/main/fdroid/repo/
https://raw.githubusercontent.com/chrisgch/tca/master/fdroid/repo
https://raw.githubusercontent.com/nucleus-ffm/Nucleus-F-Droid-Repo/master/fdroid/repo
https://raw.githubusercontent.com/rafaelvenancio98/fdroid/master/fdroid/repo
https://raw.githubusercontent.com/simlar/simlar-fdroid-repo/master/fdroid/repo
https://raw.githubusercontent.com/xarantolus/fdroid/main/fdroid/repo
https://redreader.org/fdroid
https://releases.nailyk.fr/repo/
https://repo.alefvanoon.xyz/fdroid/repo/
https://repo.beli3ver.de/
https://repo.librechurch.org/fdroid
https://repo.mobilsicher.de/fdroid/repo/
https://repo.the-sauna.icu
https://repo.unifiedpush.org/
https://s3.amazonaws.com/storymaker/fdroid/repo
https://service.tagesschau.de/app/repo/
https://skyjake.github.io/fdroid/repo
https://srv.tt-rss.org/fdroid/repo
https://submarine.strangled.net/fdroid/repo/
https://tachi.efreakbnc.net/fdroid/repo
https://thecapslock.gitlab.io/fdroid-patched-apps/fdroid/repo/
https://thedoc.eu.org/fdroid/repo
https://theplexrepo.gitlab.io/fdroid/repo
https://uget.heartshine.xyz/fdroid/repo
https://unofficial-protonmail-repository.gitlab.io/unofficial-protonmail-repository/fdroid/repo
https://update.invizbox.com/fdroid/repo
https://www.cromite.org/fdroid/repo
https://www.droidware.info/fdroid/repo
https://www.kontalk.org/fdroid/repo
https://www.ma-nic.de/fdroid/repo/
https://www.xapek.org/~yvesf/public/fdroid/repo
```

**Comment by machiav3lli** [comment]
All tested and were usable, were now added in 1.0.0
